University-advisor-web/Models/UniversityReviewModel.cs

```diff
@@ -39,7 +43,21 @@ public void SaveReviews()
 
         public bool IsDuplicate()
         {
-            return SqlDriver.Exists($"SELECT * FROM universityReviews WHERE userId ={UserId} AND universityId={UniversityId}");
+            if(SqlDriver.Exists($"SELECT * FROM universityReviews WHERE userId ={UserId} AND universityId={UniversityId}"))
+            {
+                DuplicateReviewEventArgs args = new DuplicateReviewEventArgs();
+                args.Message = Messages.reviewAlreadySubmitted;
+                args.Id = UniversityId;
+                DuplicateReview?.Invoke(this, args);
+                return true;
+            }
+            return false;
+        }
+
+        public class DuplicateReviewEventArgs : EventArgs
```

Comment:
Is this class intentionally being left here?

Reply:
yes